### ISSUE TITLE
Data set structure mismatch

### ISSUE THREAD

**Issue by farooqzaman1** [ZQPei/deep_sort_pytorch#145]
Hi, you  used data Loader from pytorch which require all images to be in separate folders each folder name should be a class name, and will contain all images of same class. All datasets you refers to are not in this format. how can i use a new dataset to retrain your model.

**Comment by Zsugabubus27** [comment]
Hi!

I also wanted to retrain the model and I have a little bit different dataset where the images are in one folder and the class of each image is encoded in the file name.
If you want to create such unique dataset, you have to create a custom dataset class in pytorch.
It is very easy if you follow this discussion:
[How to load Images without using ‘ImageFolder’](https://discuss.pytorch.org/t/how-to-load-images-without-using-imagefolder/59999)

**Comment by WurmD** [comment]
+1 to this issue.

I expect the author transformed the Market1501 dataset to necessary folder structure with a python script. The addition of this script to the repository would be appreciated.

**Comment by joaquinCaceres** [comment]
At this link is the MARS dataset in the correct format. The problem is that when training, the accuracy on the validation set in the time 20 does not exceed 1%
